t/proto_template/.github/workflows/ci.yml

```diff
@@ -73,12 +73,8 @@ jobs:
       - name: 'Run Build'
         shell: 'bash'
         run: |
-          set +e
-          BUILD_CHECK=$(buf build)
-          if ! [[ -z "${BUILD_CHECK}" ]]; then
-            echo "::error::$BUILD_CHECK"
-            exit 1
-          fi
+          set -eEuo pipefail
```

Comment:
Since you are using `shell: 'bash'` this shouldnt be needed in this workflow. The generate script you are calling however was creating a new shell which needed those set.

More info: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell

Reply:
Throughout....